lib/fireblocks/api/transactions.rb

```diff
@@ -61,21 +61,45 @@ def from_vault_to_vault(
           destination_id:,
           options: {}
         )
+          destination_params = destination_payload(
+            destination_type: 'VAULT_ACCOUNT',
+            destination_id: destination_id
+          )
+
           body = {
             amount: amount,
             assetId: asset_id,
             source: {
               type: 'VAULT_ACCOUNT',
               id: source_id
-            },
-            destination: {
-              type: 'VAULT_ACCOUNT',
-              id: destination_id
             }
-          }.merge(options)
+          }.merge(options, destination_params)
 
           create(body)
         end
+
+        def destination_payload(**kwargs)
+          if kwargs[:destinations].is_a?(Array)
+            {
+              destinations: kwargs[:destinations]
+            }
+          elsif kwargs[:destination].is_a?(Hash)
+            {
+              destination: kwargs[:destination]
+            }
+          else
```

Comment:
is this ever possible?

Reply:
Not currently, but I am [building the `destination` object for the multiple destinations](https://github.com/fundamerica/prime_trust_api/pull/2420/files#diff-4bc6339445cbeceab4d94a50d848b2f1702788a5afa72da430d4f6983f3ff2d0R13), so I added this option if we wanted to leverage the object in the future.